telegram/message.py

```diff
@@ -873,17 +874,16 @@ def parse_caption_entities(self, types=None):
             for entity in self.caption_entities if entity.type in types
         }
 
-    def _text_html(self, urled=False):
-        entities = self.parse_entities()
-        message_text = self.text
+    def _parse_html(self, text, entities, urled=False):
```

Comment:
Can you change this to be `@staticmethod`?

Reply:
why again?